wmn-data.json

```diff
@@ -6432,6 +6432,16 @@
         "m_code" : 404,
         "known" : ["carlito", "taylor"],
         "cat" : "social"
+      },
+      {
+        "name" : "Bento",
+        "uri_check" : "https://bento.me/{account}",
+        "e_code" : 200,
+        "e_string" : "Create Your Bento",

```

Comment:
Please consider changing this to something more unique like `createdAt\":`

Reply:
Updated to

```
      {
        "name" : "Bento",
        "uri_check" : "https://bento.me/{account}",
        "e_code" : 200,
        "e_string" : "href=\"https://bento.me/explore\">",
        "m_string" : ">Available!</div>",
        "m_code" : 404,
        "known" : ["carlito", "taylor"],
        "cat" : "social"
      }
```